testing/compatibility/azure/readme.md

```diff
@@ -40,11 +40,12 @@ Ensure the variables in `deploy.ps1` are correct. Some variables of note:
 
 ### Deploy
 Now, install the VMs and install Tentacle:
-> ./deploy.ps1 -octopusServerThumbprint {OctopusServerThumbprint} -octopusServerUrl {OctopusServerUri} -octopusServerApiKey {OctopusServerApiKey} -adminPassword qwerty123!@#
+> ./deploy.ps1 -octopusServerThumbprint {OctopusServerThumbprint} -octopusServerUrl {OctopusServerUri} -octopusServerApiKey {OctopusServerApiKey} -adminPassword {superSecretPasswordToTheVms.2}
```

Comment:
Why the `.2` at the end of `superSecretPasswordToTheVms.2`?

Reply:
Password requirements of needing a symbol and number.